cmd/release-notes/main.go

```diff
@@ -149,8 +149,21 @@ func init() {
 	cmd.PersistentFlags().StringVar(
 		&opts.Format,
 		"format",
-		util.EnvDefault("FORMAT", "markdown"),
-		"The format for notes output (options: markdown, json)",
+		util.EnvDefault("FORMAT", notes.FormatOutputMarkdown),
+		fmt.Sprintf("The format for notes output (options: %s)",
+			strings.Join([]string{
+				notes.FormatOutputJSON,
+				notes.FormatOutputMarkdown,
+			}, ", "),
+		),
+	)
+
+	// template is the file path to a markedown template to render notes in.
+	cmd.PersistentFlags().StringVar(
+		&opts.TemplatePath,
+		"markdown-template",
```

Comment:
I think we can make it more general and name it _go template_, right? (see comment above as link).

Reply:
I think what's meant here is "the template for the release note markdown thing". The fact that the language used by that template is the go templating language is a different aspect. I think `release-notes-template-path` or such might mouth-full but maybe be a feasible name?

Reply:
Please see next commit